README.md

```diff
@@ -51,7 +51,7 @@
         await signOut() // Sign out the user
         ```
 
-There's more supported methods in the `useSession` composable, you can create client- and server-side authentication middlewares for your app and more - read the documentation below.
+There's more supported methods in the `useSession` composable, you can create APP and API authentication middleware - read the documentation below.
```

Comment:
Two things:
- I'd either use `App` or `Application` here -> you do the same below
- Where is it established that "App" is the right name for what we called client-side before, I'm not aware of App meaning: Client-/Browser-side behavior

Reply:
- Then use Application
- You need to understand that the middleware/ directory is also called when server-rendering the page, this is why using client-side is wrong IMO

Reply:
Alright, then let's use Application.

I get it, another idea that may be clearer is to call them "universal- and server-middlewares"? This is just a nit though, so you decide which way to go here. 

If you're fine with any I may merge the PR later and then do the xhange myself to unblock you here / require less of your attention.